### DISCUSSION TITLE
Can i whitelist urls on https://dashboard.web3auth.io?

### DISCUSSION THREAD

**Discussion by billyjacoby** [Web3Auth/Web3Auth#1081] [Help]
I'm setting up branch deploys for a client side only application and can't find a way to get wildcard subdomains working properly via the web3auth dashnboard.

I found this thread that suggests using manual origin whitelisting, however I can't use the secret in my client side only application.
https://github.com/orgs/Web3Auth/discussions/351

Is there any way to add a wildcard subdomain via the web3auth dashboard?

**Comment by shahbaz17** [comment]
Hey @billyjacoby 

The Wildcard subdomain via the web3auth dashboard is not supported.

**Reply by refi93** [reply]
> Hey @billyjacoby
> 
> The Wildcard subdomain via the web3auth dashboard is not supported.

@shahbaz17 is it possible to whitelist a wildcard url by any other way, e.g. via an API or customer support request?

**Comment by wesleyfsmith** [comment]
I'd like to second the requestion for wildcarding subdomains, important for the way we do QA with vercel.
